tests/Doctrine/Tests/DBAL/Functional/Schema/SqliteSchemaManagerTest.php

```diff
@@ -232,4 +232,63 @@ public function getDiffListIntegerAutoincrementTableColumnsData()
             array('bigint', true, true),
         );
     }
+
+    /**
+     * @group DBAL-2921
+     */
+    public function testPrimaryKeyAutoIncrement()
```

Comment:
@TimoBakx thank you for the patch.

There doesn't seem to be anything SQLite-specific in this test which is good. Could you try moving it to a platform-agnostic test case (e.g. `SchemaManagerFunctionalTestCase`) to make sure it works the same on all platforms which support autoincrement fields?

Reply:
Good idea. It took a bit of tweaking, but it now runs (and passes) on all tests.